flink-runtime/src/main/java/org/apache/flink/runtime/clusterframework/ContaineredTaskManagerParameters.java

```diff
@@ -0,0 +1,182 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.runtime.clusterframework;
+
+import org.apache.flink.configuration.ConfigConstants;
+import org.apache.flink.configuration.Configuration;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * This class describes the basic parameters for launching a TaskManager process.
+ */
+public class ContaineredTaskManagerParameters implements java.io.Serializable {
+
+	private static final long serialVersionUID = -3096987654278064670L;
+	
+	/** Total container memory, in bytes */
+	private final long totalContainerMemoryMB;
+
+	/** Heap size to be used for the Java process */
+	private final long taskManagerHeapSizeMB;
+
+	/** Direct memory limit for the Java process */
+	private final long taskManagerDirectMemoryLimitMB;
+
+	/** The number of slots per TaskManager */
+	private final int numSlots;
+	
+	/** Environment variables to add to the Java process */
+	private final HashMap<String, String> taskManagerEnv;
+
+	
+	public ContaineredTaskManagerParameters(
+		long totalContainerMemoryMB, long taskManagerHeapSizeMB,
```

Comment:
I think I would place every parameter in one line or on every line multiple parameters. I don't think that multiple parameters on the first line followed by a single parameter on each following line is the most intuitive pattern.